test/Sema/implementation-only-import-suggestion.swift

```diff
@@ -34,11 +34,11 @@
 // RUN:   -library-level other -D PUBLIC_IMPORTS
 #if PUBLIC_IMPORTS
 import PublicSwift
-import PrivateSwift // expected-warning{{private module 'PrivateSwift' is imported publicly from the public module 'main'}}
+import PrivateSwift // expected-error{{private module 'PrivateSwift' is imported publicly from the public module 'main'}}
 
 import PublicClang
-import PublicClang_Private // expected-warning{{private module 'PublicClang_Private' is imported publicly from the public module 'main'}}
-import FullyPrivateClang // expected-warning{{private module 'FullyPrivateClang' is imported publicly from the public module 'main'}}
+import PublicClang_Private // expected-error{{private module 'PublicClang_Private' is imported publicly from the public module 'main'}}
+import FullyPrivateClang // expected-error{{private module 'FullyPrivateClang' is imported publicly from the public module 'main'}}
```

Comment:
I *think* you may need to mark this test as `REQUIRES: asserts` to make sure it doesn't fail on the no-asserts bots

Reply:
That makes sense, thank you!